test/test_foreach.py

```diff
@@ -519,13 +519,17 @@ def test_reduce_op(self, device, dtype, op, is_fastpath):
     @dtypes(*all_types_and_complex_and(torch.half, torch.bfloat16, torch.bool))
```

Comment:
You'll need to remove `bool` here, otherwise foreach will fail

Reply:
Got it! That was from https://github.com/pytorch/pytorch/pull/113262